zed/src/main.cpp

```diff
@@ -247,17 +250,15 @@ class zed_imu_thread : public threadloop {
     // IMU
     SensorsData sensors_data;
     Timestamp last_imu_ts = 0;
-    Eigen::Vector3f la;
-    Eigen::Vector3f av;
-
-    // Timestamps
-    time_type t;
-    ullong imu_time;
 
     std::size_t last_serial_no {0};
 
     // Logger
     record_coalescer it_log;
+
+	std::optional<ullong> _m_first_imu_time;
+	std::optional<time_point> _m_first_real_time;
```

Comment:
Same as realsense. 

> These are unset when the object is constructed, and set during the first execution of callback.

> Using a std::optional<...> is better than using a sentinel value, such as 0, because you don't have to assume things about the domain of the variable.